.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
ERROR:C:\Users\Irina\AndroidStudioProjects\study\FirstNotebook\app\src\main\res\layout\activity_edit.xml:16: AAPT: error: resource menu/toolbar_btns (aka com.example.practice:menu/toolbar_btns) not found.